include/picongpu/plugins/shadowgraphy/Shadowgraphy.hpp

```diff
@@ -251,15 +251,18 @@ namespace picongpu
                 {
                     return std::shared_ptr<plugins::multi::IHelp>(new Help{});
                 }
-
-                //! Implementation of base class function.
-                void notify(uint32_t currentStep) override
+                .
```

Comment:
BUG, someone slept on the keyboard!

Reply:
Oops. I should have run the test on the "only documentation commit" as well

Reply:
Thanks for finding that!